Source/Fuse.Nodes/Visual.Layout.uno

```diff
@@ -417,9 +421,9 @@ namespace Fuse
 		protected virtual float2 OnArrangeMarginBox(float2 position, LayoutParams lp)
 		{
 			var sz = float2(0);
-			for (int i=0; i < ZOrderChildCount; ++i)
+			for (var v = FirstChild<Visual>(); v != null; v = v.NextSibling<Visual>())
 			{
-				var msz = GetZOrderChild(i).ArrangeMarginBox(position, lp);
+				var msz = v.ArrangeMarginBox(position, lp);
```

Comment:
Though this change is okay I wonder if we should mention it in the CHANGELOG. It may have an effect on elements that have layout relative to each other, like `LayoutMaster` or fucntions `width` and `height`. Those features are *supposed* to not care about ordering, but maybe we missed a spot.